### PR TITLE
Fix warnings in documentation

### PR DESCRIPTION
**Pull request by rw1nkler**
This commit resolves all warnings in the project documentation.

### CODE REVIEW

File: docs/db_dev_process/fuzzers/index.rst [hunk 1]
```diff
@@ -57,7 +57,6 @@ Programmable Interconnect Points (PIPs)
    :maxdepth: 1
```
Comment (mithro) [inline docs/db_dev_process/fuzzers/index.rst]: Why are these others being removed?

File: docs/conf.py [hunk 2]
```diff
@@ -100,7 +100,10 @@
 # List of patterns, relative to source directory, that match files and
 # directories to ignore when looking for source files.
 # This patterns also effect to html_static_path and html_extra_path
-exclude_patterns = ['_build', 'Thumbs.db', '.DS_Store']
+exclude_patterns = [
+    '_build', 'architecture/copying.md', 'db_dev_process/minitests/index/**',
```
Comment (mithro) [inline docs/conf.py]: One per line would be better.

File: docs/db_dev_process/fuzzers/index.rst (hunk 1 above)
Comment (rw1nkler) [reply docs/db_dev_process/fuzzers/index.rst]: Those fuzzers do not have `README.md` file.

File: docs/conf.py (hunk 2 above)
Comment (rw1nkler) [reply docs/conf.py]: It was like that, but the format auto-check on Travis complained about the formatting.
I pushed again the previous version.

**Comment by rw1nkler** [comment]
I restored the previous version obtained using `make format` command. This ensures that the formatting is valid in CI.

One per line formating is not allowed by CI, @mithro:
format1
format2

**Comment by tmichalak** [comment]
@mithro Do we merge this one now and then open an issue to fix the formatter?

**Comment by mithro** [comment]
You should be able to add markers to the file to tell yapf to ignore this structure.

**Comment by rw1nkler** [comment]
@mithro, I've added the yapf markers

**Comment by rw1nkler** [comment]
Could we merge this? @mithro

**Pull request merged**